hack/ci/e2e.sh

```diff
@@ -122,7 +122,7 @@ run_tests() {
     local fixed_coredns
     fixed_coredns=$(
       sed \
-        -e 's/^(\s)+.*cluster\.local.*$/\0kubernetes cluster.local internal in-addr.arpa ip6.arpa/' \
```

Comment:
Internal is right here...?

Reply:
Is after cluster.local because prow uses that domain, if doesn't resolve, because we don't have upstream DNS servers, it answer no exist and the pod stops trying

Reply:
No I mean, the existing replace includes internal 🤷‍♂️